openfisca_us/parameters/fcc/lifeline/amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Maximum Lifeline benefit amount
+values:
+  2016-12-01: 9.25
+metadata:
+  period: month
+  unit: currency-USD
+  reference:
+    - title: 47 CFR § 54.409 - Consumer qualification for Lifeline.
```

Comment:
Found elsewhere -I'd favour just referencing the $9.25 in the legislation: https://www.law.cornell.edu/cfr/text/47/54.403

Reply:
And in the references on that page, looks like it came into effect on April 2, 2012: https://www.govinfo.gov/content/pkg/FR-2012-03-02/pdf/2012-4978.pdf#page=17

Reply:
The first link also has the historical rates (7.25, 5.25 and beyond)

Reply:
Thanks - updated the reference and added the 77 FR 12967 source for the effective date. The 7.25/5.25 are rates for phone-only service. For now I'm entering the full basic rate, which includes broadband, especially since standalone phone service is no longer covered. But filed #408 to look into this in the future.